seed/lib/merging/merging.py

```diff
@@ -124,11 +125,22 @@ def merge_state(merged_state, state1, state2, priorities, ignore_merge_protectio
     # Calculate the difference between the two states and save into a dictionary
     can_attrs = get_state_attrs([state1, state2])
 
+    recognize_empty_columns = state2.organization.column_set.filter(
+        table_name=state2.__class__.__name__,
+        recognize_empty=True,
+        is_extra_data=False
+    ).values_list('column_name', flat=True)
+
     default = state2
     for attr in can_attrs:
-        # Do we have any differences between these fields? - Check if not None instead of if value.
-        attr_values = [value for value in list(can_attrs[attr].values()) if value is not None]
-        attr_values = [v for v in attr_values if v is not None]
```

Comment:
I understood these two lines to be doing the same thing. Let me know if I'm misunderstanding something.

Reply:
yeah, those look redundant... clean up later though?

Reply:
I ended up replacing those with a single list comprehension (that now incorporates the "recognize_empty" setting), so I think we're good here.